frontend/src/lib-components/atoms/form/InputField.tsx

```diff
@@ -283,9 +283,8 @@ export const TextArea = styled(TextareaAutosize)`
   resize: none;
 
   background-color: transparent;
-  border-style: solid;
-  border-width: 0 0 1px;
-  border-color: ${({ theme: { colors } }) => colors.greyscale.medium};
+  border: 2px solid transparent;
+  border-bottom: 1px solid ${({ theme: { colors } }) => colors.greyscale.medium};
```

Comment:
Do these changes affect many parts of the project?

Reply:
Yes, and I actually checked that the new implementation fixes the flickering effect in FF/Safari/Chrome.